_data/packages.yml

```diff
@@ -368,7 +408,7 @@
       github_username: magsol
     - name: Jakub Tomasz Gnyp
       github_username: gnypit
-  archive: '[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7268074.svg)](https://doi.org/10.5281/zenodo.7268074)'
+  archive: '[10.5281/zenodo.7268074](https://doi.org/10.5281/zenodo.7268074)'
```

Comment:
Here pyosmeta is making a decision about how doi's are processed. i opened an issue [here about this](https://github.com/pyOpenSci/pyosMeta/issues/228)

i think it's pretty easy for a user or maintainer or editor to copy a doi badge into the issue. and it renders nicely too. so that could be a way for us to go. 

pyosmeta can them parse instances like this where the badge part is missing accordingly. the website can then render as needed based on the badge. right now it's parsing the badge into html. 

we could decide to go the opposite way too but whatever we decide it needs to be consistent.